modules/rn_3_7_0.adoc

```diff
@@ -0,0 +1,67 @@
+[[rn-3-700]]
+== Version 3.7.0
+
+=== quay / clair / quay-builder
+
+Added/Changed:
+
+* With {productname} 3.7, users have the ability to report storage consumption and to contain registry growth by establishing configured storage quota limits. Although there are no limitations on a Quay organization's storage consumption, on-premise Quay users are now equipped with quota reporting and quota management capabilities to manage the capacity limits of their environment. (link:https://issues.redhat.com/browse/PROJQUAY-302[PROJQUAY-302], link:https://issues.redhat.com/browse/PROJQUAY-253[PROJQUAY-253])
```

Comment:
> Although there are no limitations on a Quay organization's storage consumption

How are limitations on an organizations storage consumption different than managing the capacity limits? I think we're offering a guarantee that an org cannot exceed the quota limit if a policy is set to reject at a certain percentage. @kwestpharedhat  that sound right?